### PR TITLE
SPCTR-3444: Fix - Buttons margin getting affected by child margin issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Buttons margin is getting changed when button-child margin is changed.
https://brainstormforce.atlassian.net/browse/SPCTR-3444

### Screenshots

https://d.pr/v/lYUTOp

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ### x.x.x ###
 * Improvement: Buttons - Added a new vertical alignment option.
+* Fix: Buttons - Resolved an issue in which the margins added to buttons is getting overridden by child-button margins.
```
Comment (bsf-zanev) [inline README.md:220]:
```suggestion
* Fix: Buttons - Resolved an issue in which the margins added to buttons group would get overridden by child button margins. 
```

File: includes/blocks/buttons/frontend.css.php [hunk 2]
```diff
@@ -154,24 +150,17 @@
...
 		'padding-left'   => UAGB_Helper::get_css_value( $attr['leftTabletPadding'], $attr['tabletPaddingUnit'] ),
 		'padding-right'  => UAGB_Helper::get_css_value( $attr['rightTabletPadding'], $attr['tabletPaddingUnit'] ),
-		'margin-top'     => UAGB_Helper::get_css_value( $attr['topMarginTablet'], $attr['marginType'] ),
-		'margin-bottom'  => UAGB_Helper::get_css_value( $attr['bottomMarginTablet'], $attr['marginType'] ),
-		'margin-left'    => UAGB_Helper::get_css_value( $attr['leftMarginTablet'], $attr['marginType'] ),
-		'margin-right'   => UAGB_Helper::get_css_value( $attr['rightMarginTablet'], $attr['marginType'] ),
 	);
 
 	$button_mobile_style = array(
+
```
Comment (bsf-zanev) [inline includes/blocks/buttons/frontend.css.php:156]: Coding Standards: This extra line is not really needed.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me, just a changelog update required.

File: includes/blocks/buttons/frontend.css.php (hunk 2 above)
Comment (rajmohanreddy774) [reply includes/blocks/buttons/frontend.css.php]: Removed.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**